src/ruby/devcontainer-template.json

```diff
@@ -9,19 +9,21 @@
     "options": {
         "imageVariant": {
             "type": "string",
-            "description": "Ruby version (use -bullseye variants on local arm64/Apple Silicon):",
+            "description": "Ruby version (use -bookworm, -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
-                "3",
-                "3.1",
-                "3.0",
+                "3-bookworm",
+				"3.2-bookworm",
+                "3.1-bookworm",
                 "3-bullseye",
+				"3.2-bullseye",
                 "3.1-bullseye",
                 "3.0-bullseye",
                 "3-buster",
+				"3.2-buster",
                 "3.1-buster",
                 "3.0-buster"
             ],
-            "default": "3-bullseye"
+            "default": "3.2-bullseye"
```

Comment:
I don't have a strong opinion, but I would say `3-bullseye` is a more natural default.  Why change to 3.2?

Reply:
Once ruby `3.3` is released, it might break things right? 

Or are we confident that ruby won't release a breaking change with major version 3? 🤔

Reply:
I suppose I don't know about about ruby to make that call, so i'm happy with 3.2 like you did 👍